static/js/hotkey.js

```diff
@@ -409,8 +409,11 @@ function handle_popover_events(event_name) {
 
 // Returns true if we handled it, false if the browser should.
 export function process_enter_key(e) {
-    if ($(".dropdown.open").length && $(e.target).attr("role") === "menuitem") {
-        // on #gear-menu li a[tabindex] elements, force a click and prevent default.
+    if (
```

Comment:
This is just adding the class `.dropup` because the dropdown added here is technically a dropup menu